docs/conf.py

```diff
@@ -12,17 +12,46 @@
 
 autodoc_mock_imports = ["f90nml", "jsonschema", "lxml"]
 copyright = str(dt.datetime.now().year)
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx"]
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.extlinks", "sphinx.ext.intersphinx"]
+extlinks_detect_hardcoded_links = True
```

Comment:
This is supposed to detect cases where a defind `extlink` could have been used in place of a hardcoded link. We don't have any of those at the moment, so we'll see if this adds any value. It currently does no harm anyway.